tests/records/full-record.json

```diff
@@ -118,12 +118,16 @@
       "lang": "eng"
     }],
     "locations": [{
-      "point": {
-        "lat": 1,
-        "lon": 2
+      "geometry": {
```

Comment:
Comment: Here's an example of with Getty TGN and a Geoname (and CERN as example)
```
"locations": [{
      "geometry": {
        "type": "Point",
        "coordinates": [6.05, 46.23333]
      },
      "identifiers": {
        "geonames": "2661235",
        "tgn": "http://vocab.getty.edu/tgn/8703679"
      },
      "place": "CERN",
      "description": "Invenio birth place."
    }],
```

Reply:
Thanks a lot! :100: